apps/web/src/components/NftPreview/NftPreview.tsx

```diff
@@ -272,7 +264,6 @@ const StyledNftPreview = styled.div<{
   position: relative;
   overflow: hidden;
   max-height: 80vh;
-  max-width: ${({ aspectRatio }) => `calc(80vh * ${aspectRatio})`};
```

Comment:
@Robinnnnn do you remeber why we set the max width here? Seems like removed it, will fix all the nft overflow issue. 

Tested on chrome, safari, firefox on big galllery profile

Reply:
hmm looks like i added that line here: https://github.com/gallery-so/gallery/pull/939

Reply:
i checked out the Curated gallery and it looks fine though, so will go ahead and merge this 👍